src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck is doing its work.");

```

Comment:
The message 'Truck is doing its work.' is not very informative. It would be better to specify what kind of work the Truck is doing, to make the message more informative as per the checklist.